.circleci/config.yml

```diff
@@ -90,3 +117,44 @@ jobs:
             mkdir -p $HOME/testground/plans
             ln -s $HOME/project/plans/placebo $HOME/testground/plans/placebo
             go test -v ./...
+
+  # Tests the k8s runner can successfully submit jobs.
+  kind-test:
+    executor: linux
+    steps:
+      - setup-install-bins
+      - run:
+          name: "build testground"
+          command: make install
+      - run:
+          name: "configure cluster"
+          command: |
+            kind create cluster --wait 90s
+            kubectl apply -f .circleci/pv.yaml
+            kubectl apply -f .circleci/pvc.yaml
+            kubectl label nodes kind-control-plane testground.nodetype=plan
+      # The placebo does not require a sidecar.
+      # To prevent kind from attempting to download the image from docker hub,
+      # build and load the image before executing it.
+      # The plan is renamed as `testplan:placebo` because kind will check dockerhub if the tag is `latest`.
+      - run:
+          name: "run a plan"
+          command: |
+            testground daemon &
+            testground plan import --from plans/placebo
+            testground build single --builder docker:go --plan placebo | tee build.out
+            export ARTIFACT=$(awk -F\" '/generated build artifact/ {print $8}' build.out)
+            docker tag $ARTIFACT testplan:placebo
+            kind load docker-image testplan:placebo
+            testground run single --runner cluster:k8s --builder docker:go --use-build testplan:placebo --instances 1 --plan placebo --testcase ok | tee run.out
+      - run:
+          name: "verify output"
```

Comment:
I had initially thought I would do --collect and verify the output captured by --collect is the same as what we see here. This does not work because I am only building a single-node k8s cluster and I have to label it as either `testground.nodetype=infra` or `testground.nodetype=plan`. If we want to test. So the collect-outputs pod waits forever a node.

I would point out that kind *does* support "multi-node" containers.
If we *do* want to test the "collection" pod behavior, I think I would just do something like this rather than a multi-node cluster.
1. set the label to `testground.nodetype=plan`
2. execute the plan, just as we already are
3. change the label to `testground.nodetype=infra`
4. test the collection path with `testground collect`

But for now I'm just checking the output by logging into the node and looking at the files.

Reply:
Instead of doing `nodetype`, can't we have labels for multiple roles?

Something like:
`testground.role.infra = true`
`testground.role.plan = true`

This way, we can tag the same node with multiple roles, and avoid having to run multiple nodes for various roles.